pkg/openstack/loadbalancer.go

```diff
@@ -467,7 +468,12 @@ func (lbaas *LbaasV2) createOctaviaLoadBalancer(name, clusterName string, servic
 	}
 
 	if svcConf.supportLBTags {
-		createOpts.Tags = []string{svcConf.lbName}
+		customTags, err := lbaas.customLoadBalancerListenerTag(service, svcConf)
+		if err != nil {
+			return nil, err
```

Comment:
Should this be fatal? Honestly I'm not 100% sure. Probably it doesn't hurt it currently is.

Reply:
fixed and removed fatality